src/promptflow/tests/executor/unittests/_core/test_tools_manager.py

```diff
@@ -100,6 +106,41 @@ def sample_tool(input: str):
     return input
 
 
+def mock_func1():
```

Comment:
add some comments here to let other devs know this is for dynamic list function testing.

Reply:
added